nifi-commons/nifi-security-utils/src/test/java/org/apache/nifi/security/util/crypto/AESKeyedCipherProviderTest.java

```diff
@@ -14,311 +14,298 @@
 
-            byte[] cipherBytes = cipher.doFinal(PLAINTEXT.getBytes("UTF-8"))
-            logger.info("Cipher text: ${Hex.encodeHexString(cipherBytes)} ${cipherBytes.length}")
+            byte[] cipherBytes = cipher.doFinal(PLAINTEXT.getBytes("UTF-8"));
 
-            cipher = cipherProvider.getCipher(em, key, iv, false)
-            byte[] recoveredBytes = cipher.doFinal(cipherBytes)
-            String recovered = new String(recoveredBytes, "UTF-8")
-            logger.info("Recovered: ${recovered}")
+            cipher = cipherProvider.getCipher(em, key, iv, false);
+            byte[] recoveredBytes = cipher.doFinal(cipherBytes);
+            String recovered = new String(recoveredBytes, "UTF-8");
 
             // Assert
-            assertEquals(PLAINTEXT, recovered)
+            assertEquals(PLAINTEXT, recovered);
         }
     }
 
     @Test
     void testGetCipherWithExternalIVShouldBeInternallyConsistent() throws Exception {
         // Arrange
-        KeyedCipherProvider cipherProvider = new AESKeyedCipherProvider()
+        KeyedCipherProvider cipherProvider = new AESKeyedCipherProvider();
 
         // Act
-        keyedEncryptionMethods.each { EncryptionMethod em ->
-            logger.info("Using algorithm: ${em.getAlgorithm()}")
-            byte[] iv = cipherProvider.generateIV()
-            logger.info("IV: ${Hex.encodeHexString(iv)}")
+        for (final EncryptionMethod em : keyedEncryptionMethods) {
+            byte[] iv = cipherProvider.generateIV();
 
             // Initialize a cipher for encryption
-            Cipher cipher = cipherProvider.getCipher(em, key, iv, true)
+            Cipher cipher = cipherProvider.getCipher(em, key, iv, true);
 
-            byte[] cipherBytes = cipher.doFinal(PLAINTEXT.getBytes("UTF-8"))
-            logger.info("Cipher text: ${Hex.encodeHexString(cipherBytes)} ${cipherBytes.length}")
+            byte[] cipherBytes = cipher.doFinal(PLAINTEXT.getBytes("UTF-8"));
 
-            cipher = cipherProvider.getCipher(em, key, iv, false)
-            byte[] recoveredBytes = cipher.doFinal(cipherBytes)
-            String recovered = new String(recoveredBytes, "UTF-8")
-            logger.info("Recovered: ${recovered}")
+            cipher = cipherProvider.getCipher(em, key, iv, false);
+            byte[] recoveredBytes = cipher.doFinal(cipherBytes);
+            String recovered = new String(recoveredBytes, "UTF-8");
 
             // Assert
-            assertEquals(PLAINTEXT, recovered)
+            assertEquals(PLAINTEXT, recovered);
         }
     }
 
     @Test
     void testGetCipherWithUnlimitedStrengthShouldBeInternallyConsistent() throws Exception {
         // Arrange
-        assumeTrue(isUnlimitedStrengthCryptoAvailable(), "Test is being skipped due to this JVM lacking JCE Unlimited Strength Jurisdiction Policy file.")
+        assumeTrue(isUnlimitedStrengthCryptoAvailable(), "Test is being skipped due to this JVM lacking JCE Unlimited Strength Jurisdiction Policy file.");
```

Comment:
As mentioned above, this check can be removed.

Reply:
Remove it